src/components/searchbar.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import { fetchMovies } from '../actions/movies';
+
+
+class Searchbar extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            s: ''
+        }
+
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(e) {
+        this.setState({
+            [e.target.dataset.queryName]: e.target.value
+        });
+    }
+
+    handleSubmit(e) {
+        this.props.fetchMovies(this.state);
+        e.preventDefault();
+    }
+
+    render() {
+        return (
+            <form onSubmit={this.handleSubmit}>
+                <label>Nazwa filmu:</label>
+                <input
+                    name="movieTitle"
+                    type="text"
+                    data-query-name="s"
```

Comment:
As we would like to keep our components as dumb ones, we don't want it to be somehow connected to details of our API. This comment quite overlaps with the self explanatory variable names. 

That will make it possible to reuse component with different API. That leads us to keep API related details in different place than in component itself. I suggest create separate const object in `constants.js` file like `API_PARAMETERS` which could contain hash map with self explanatory options with values referring to the API itself ie. 
```javascript
API_PARAMETERS: {
  SEARCH: 's'
}
```